torch/cuda/memory.py

```diff
@@ -179,6 +180,12 @@ def empty_cache() -> None:
         torch._C._cuda_emptyCache()
 
 
+def enable_cache(value: bool = True) -> None:
```

Comment:
set_cache_allocator, it's a bit weird to call "enable_cache(False)" to disable the cache allocator.

Reply:
It might seem weird, but there is some precedent already in this file.